src/main/java/adapters/default_game/GamePanelPresenter.java

```diff
@@ -23,13 +23,27 @@ public class GamePanelPresenter extends JPanel implements IGamePanelOutputBounda
     private int playerX;
     private int playerY;
 
+    private final MazeHazards hazards;
+    private final MazeItems items;
+
     /**
      * Construct a new GamePanelPresenter with fixed settings.
      **/
-    public GamePanelPresenter() {
+    public GamePanelPresenter(MazeItems items, MazeHazards hazards) {
```

Comment:
What would the OutputBoundary for items_hazards include? I know for UpdateMaze it's playerX/Y. 

Actually I think Player also should implement "draw" so it can just use IDrawOutputBoundary instead. Right now the reason why you have a separate method UpdateMaze is to draw the white rectangle. But why not just use IDrawOutputBoundary?
also wanna see what @ltenenb thinks